### PR TITLE
Add terms for bulk reports

### PR DESCRIPTION
**Pull request by olovy**

### CODE REVIEW

File: source/vocab/details.ttl [hunk 1]
```diff
@@ -806,7 +807,7 @@
     rdfs:label "start time"@en, "starttid"@sv.
 
 :endTime a owl:DatatypeProperty ;
-    rdfs:domain :MediaObject ; #for event and bitstream?
+    sdo:domainIncludes :MediaObject, bulk:Execution ; #for event and bitstream?
```
Comment (niklasl) [inline source/vocab/details.ttl:810]: +1 we should "appropriately" widen some of these domains.

File: source/vocab/display.jsonld [hunk 2]
```diff
@@ -586,6 +586,12 @@
           "@type": "fresnel:Lens",
           "classLensDomain": "License",
           "showProperties": [ "title", "identifier" ]
+        },
```
Comment (kwahlin) [inline source/vocab/display.jsonld:589]: Copy-pasted? :slightly_smiling_face:

**Review by kwahlin — CHANGES_REQUESTED** [review]
See comment. Otherwise LGTM!

File: source/vocab/display.jsonld [hunk 3]
```diff
@@ -586,6 +586,12 @@
...
+        "bulk:Execution": {
+          "@id": "License-chips",
+          "@type": "fresnel:Lens",
+          "classLensDomain": "License",
+          "showProperties": [ "bulk:status" ]
```
Comment (kwahlin) [inline source/vocab/display.jsonld:594]: Copy-pasted? :slightly_smiling_face:

Comment (olovy) [reply source/vocab/display.jsonld:594]: Yes! :pray:

Comment (olovy) [inline source/vocab/display.jsonld:594]:
```suggestion
        "bulk:Execution": {
          "@id": "bulk:Execution-chips",
          "@type": "fresnel:Lens",
          "classLensDomain": "bulk:Execution",
          "showProperties": [ "bulk:status" ]
```

**Pull request merged**